src/base/ApplicationECSService.ts

```diff
@@ -389,6 +398,17 @@ export class ApplicationECSService extends Resource {
         def.mountPoints.forEach((mountPoint) => {
           // We currently only set the volume names, but more configuration is available in EcsTaskDefinitionVolume.
           volumes[mountPoint.sourceVolume] = { name: mountPoint.sourceVolume };
+          if (
+            this.config.efsConfig &&
+            this.config.efsConfig.volumeName == mountPoint.sourceVolume
```

Comment:
I'm not sure what type (string maybe?) is `mountPoint.sourceVolume` but shouldn't we be using `===` instead of `==` 🤔

Reply:
Updated pr with this suggestion